### ISSUE TITLE
Open source doltlab and dolthub

### ISSUE THREAD

**Issue by Igorgro** [dolthub/dolt#5133]
There were some plans to open source doltlab and dolthub in 2022. What is the current state of this? I cannot find any further information

**Comment by coffeegoddd** [comment]
@Igorgro you're correct, we did tease and discuss open-sourcing both DoltLab and DoltHub in 2022. It's also been requested quite often. @timsehn for visibility.

To my knowledge, we've decided to hold off on open-sourcing both products, at least through Q1 2023, since this would be a heavy engineering lift.

If you're open to discussing your use-case for DoltLab/DoltHub we'd be happy to work with you to ensure each product meets your needs.

**Comment by timsehn** [comment]
This is a heavy technical lift. I'll move this issue to be tracked in the doltlab-issues repo.
